gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -363,7 +363,7 @@ func (vm *VMKeeper) Run(ctx sdk.Context, msg MsgRun) (res string, err error) {
 		m2.Release()
 	}()
 	m2.RunMain()
-	ctx.Logger().Info("CPUCYCLES call: ", m2.Cycles)
+	ctx.Logger().Info("CPUCYCLES call: ", "cycles", m2.Cycles)
```

Comment:
```suggestion
	ctx.Logger().Info("CPUCYCLES call",
		 "cycles", m2.Cycles,
	)
```

Not sure, I like both formats.

Reply:
Good catch 💯 

Applied the suggestion:

[34e4087](https://github.com/gnolang/gno/pull/1302/commits/34e40874a10e886cbbbab82102334272ac15b434)

Reply:
Can't we have something like logrus
```go
ctx.Logger().WithFields(slog.Fields{
    "cycles", m2.Cycles,
}).Info("CPUCYCLES call")
```

and/or

```go
ctx.Logger().WithField("cycles", m2.Cycles).Info("CPUCYCLES call")
```

Reply:
Hmm, I don't understand why logrus is considered superior. The `slog` API is extremely straightforward.